docs/docs/12-context.md

```diff
@@ -160,6 +160,35 @@ function Button(props, context) {
 Button.contextTypes = {color: React.PropTypes.string};
 ```
 
+## How to context updates
+
+The `getChildContext` function will be called when the state or props changes. In order to update data in the context, trigger a local state update with `this.setState`. This will trigger a new context and changes will be received by the children.
+
+```javascript
+var MediaQuery = React.createClass({
+  getInitialState(){
+    return {type:'desktop'};
+  },
+  childContextTypes: {
+    type: React.PropTypes.string
+  },
+  getChildContext: function() {
+    return {type: this.state.type};
+  },
+  componentDidMount(){
+    var checkMediaQuery = function(){
+      var type = window.matchMedia("min-width: 1025px").matches ? 'desktop' : 'mobile';
+      if (type !== this.state.type){
+        this.setState({type:type}); 
+      }
+    };
+    
+    window.addEventListener('resize', checkMediaQuery);
+    checkMediaQuery();
+  }
+});
```

Comment:
Also, we need a `render` function. You can keep it simple and just render something like `<ComponentThatUsesContext />`